lib/msal-core/src/Constants.ts

```diff
@@ -133,6 +141,19 @@ export const PromptState = {
 	NONE: "none",
 };
 
+/**
+ * @hidden
+ */
 export const Library = {
   version: "1.0.2"
 };
+
+/**
+ * @hidden
+ * Readable shortcut for Interaction Required Errors
+ */
+export const InteractionErrorType = {
```

Comment:
could this be made an enum?

Reply:
More for consistency sake. I can push a PR right after making enums the NORM in constants file if that is what we want to do. I do not want to break consistency, it makes the code look ugly if you see what I mean :)